.github/workflows/CI.yml

```diff
@@ -133,6 +133,10 @@ jobs:
           wget https://get.helm.sh/helm-v3.1.1-linux-386.tar.gz && tar zxvf helm-v3.1.1-linux-386.tar.gz && \
           sudo mv linux-386/helm /usr/local/bin/helm3 && \
           helm3 plugin install https://github.com/chartmuseum/helm-push
+          git clone https://github.com/cnabio/cnab-to-oci.git
```

Comment:
Could we use a fixed release version, in case the master branch may break our testing

Reply:
Yes, it's pinned to the latest version v0.3.0-beta4 now.